src/Client.cpp

```diff
@@ -183,6 +210,9 @@ void Client::OnAfterCreated(CefRefPtr<CefBrowser> browser)
 
     CEF_REQUIRE_UI_THREAD();
 
+    m_requestHandler->m_messageRouterBrowserSide = CefMessageRouterBrowserSide::Create(constants::messageRouterConfig);
```

Comment:
I'm afraid concurrent calls to OnAfterCreated would override this message router. `cef-pdf` is designed to allow many requests at the same time